app/controllers/users_controller.rb

```diff
@@ -113,6 +113,16 @@ def extlogout
     render :extlogout, :layout => 'login'
   end
 
+  def test_mail
+    user = find_resource(:edit_users)
+    begin
+      MailNotification[:tester].deliver(:user => user)
+    rescue Net::SMTPAuthenticationError, Net::SMTPServerBusy, Net::SMTPSyntaxError, Net::SMTPFatalError, Net::SMTPUnknownError, Timeout::Error => e
+      render :json => e.message, :status => :internal_server_error and return
+    end
+    render :json => {:message => _("Email was sent successfully")}, :status => :ok
+  end
```

Comment:
@lzap Thanks for the review, I added more exceptions which may be related to  `sendmail` , do you know about more / others ?